src/App.tsx

```diff
@@ -1,9 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(({ title, description }) => {
+    const lowerTitle = title.toLowerCase();
+    const lowerDescription = description.toLowerCase();
+    const lowerQuery = query.toLowerCase();
+
+    return lowerTitle.includes(lowerQuery.trim())
+      || lowerDescription.includes(lowerQuery.trim());
+  });
```

Comment:
you can take formatted query declaration out of the callback for better performance

Reply:
Thanks I will do)